src/main.cpp

```diff
@@ -19,12 +22,38 @@
 #include <domains/dom_runner.h>
 #endif
 
+#ifdef CONFIG_AOS_MOUNT_LFS_MMC
```

Comment:
Could it be done as automount through config? If not I don't like to bloat main with low level code. As proper solution, move related code to `bsp` folder into `mount.c/h` for example.

Reply:
Sadly, it can only be done with code. I pushed a new version where mounting is moved to storage.cpp